waiter/integration/waiter/authentication_test.clj

```diff
@@ -121,6 +121,11 @@
                                                                  :headers {"content-type" "application/x-www-form-urlencoded"}
                                                                  :method :post)
             _ (assert-response-status response 303)
+            cookie-fn (fn [cookies name] (some #(when (= name (:name %)) %) cookies))
+            auth-cookie (cookie-fn cookies "x-waiter-auth")
+            _ (is (not (nil? auth-cookie)))
+            one-hour (* 60 60)
```

Comment:
Nitpick: `one-hour-in-secs (-> 1 t/hour t/in-seconds)`, can even inline the variable in the assertion below